spec/score_card_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'score_card'
+
+describe ScoreCard do
+  let(:bowl) { ScoreCard.new }
+  let(:frame) { double :frame }
```

Comment:
At the moment, I think this line is unnecessary as you are not using a double in the tests - the methods interacting with frame objects are calling actual frame objects directly.